github/repos_traffic_test.go

```diff
@@ -308,3 +309,18 @@ func TestTrafficData_Marshal(t *testing.T) {
 
 	testJSONMarshal(t, u, want)
 }
+
+func TestTrafficBreakdownOptions_Marshal(t *testing.T) {
+	testJSONMarshal(t, &TrafficBreakdownOptions{}, "{}")
+	v, _ := url.Parse("day")
+
+	u := &TrafficBreakdownOptions{
+		Per: v.String(),
```

Comment:
Since this is just a simple string, let's please get rid of line 315 above and replace this with:

```suggestion
		Per: "day",
```